azimuth/modules/base_classes/dask_module.py

```diff
@@ -78,9 +79,8 @@ def get_caching_indices(self) -> List[int]:
         raise NotImplementedError
 
     @property
-    def task_id(self) -> Tuple[str, int]:
-        indices_hash = hash(tuple(self.get_caching_indices()))
-        return self.name, indices_hash
+    def task_id(self) -> str:
+        return "_".join(map(str, (self.name, hash(tuple(self.get_caching_indices())))))
```

Comment:
Any reason for keeping the `map()` here? I think it's rather confusing.
```suggestion
        return f"{self.name}_{hash(tuple(self.get_caching_indices()))}"
```

Reply:
Right, I agree. I had something ish similar but `mypy` was complaining. 
`return self.name+"_"+str(hash(tuple(self.get_caching_indices())))}`.

But the f-string is much cleaner and `mypy` shouldn't complain. Thanks!

Reply:
Right! Since f-strings implicitly calls `str()` on things, so you don't have to do it yourself.

Reply:
Oh, right!!